rootfs-builder/rootfs.sh

```diff
@@ -264,6 +279,11 @@ check_env_variables()
 
 	[ "$AGENT_INIT" == "yes" -o "$AGENT_INIT" == "no" ] || die "AGENT_INIT($AGENT_INIT) is invalid (must be yes or no)"
 
+	if [ -z "${AGENT_SOURCE_BIN}" ]; then
```

Comment:
should this be a check for `${RUST_AGENT}`, instead of `${AGENT_SOURCE_BIN}` ?

Reply:
It's ${AGENT_SOURCE_BIN}., which is the built binary